server/routes/api.js

```diff
@@ -115,31 +119,40 @@ routes.get('/pairing/paired', (req, res) => {
   });
 });
 
-routes.get('/wishlist/:familyId', (req, res) => {
+routes.patch('/wishlist/:familyId/addItem', wishlistController.addItem);
+
+routes.patch('/wishlist/:familyId/create', wishlistController.create);
+
+routes.patch('/wishlist/:familyId/removeItem', wishlistController.removeItem);
+
+routes.patch('/wishlist/:familyId/updateItem', wishlistController.updateItem);
 
+routes.get('/wishlist/:familyId', (req, res) => {
```

Comment:
from an optimization standpoint, I *think* this route should go before the routes built off this, e.g. `/wishlist/:familyId/updateItem`